CHANGELOG.md

```diff
@@ -1,8 +1,28 @@
-# Development
+# 1.0.0
 
-* Rename fixtures to examples (breaking change – all documentation will need updating when upgrading)
+All documentation will need updating when upgrading (see PR #37):
+* Rename fixtures to examples in YAML documentation
 * Nest example data within a named block to give examples more features
 
+Steps to upgrade:
+* Rename fixtures in YAML files to "examples"
+* Nest data for each example in a `data:` block
```

Comment:
Maybe bit picky, but I feel like the last 2 bullet points just re-word the 2 bullet points before it - are they both needed?

Reply:
Probably not but maybe not worth blocking on?